### DISCUSSION TITLE
`Switch` not exported from `react-router-dom` or compat package

### DISCUSSION THREAD

**Discussion by baker-travis** [remix-run/react-router#8754] [v5 to v6 Migration] [answered]
The migration guide shows that we should be able to keep our existing `Switch` in, and then when we've converted all routes to v6, move it over to `Routes`. However, `Switch` isn't exported from `react-router-dom@6.3.0` or `react-router-dom-v5-compat@6.3.0`. What do we need to do in order to keep our existing `Switch` in tact?

**Comment by diondiondion** [comment, marked as answer]
Keep importing it from react-router-dom v5. You shouldn't have changed the react-router-dom dependency to 6.3.0 before you've migrated all `Switch` components to `Routes`.

**Reply by scrozier** [reply]
I get a peer dependency issue when trying to install react-router-dom@^5.2.0 and react-router-dom-v5-compat@6.4.1. Can you advise on that situation?

**Reply by ryanhiebert** [reply]
Same for me.

**Reply by ryanhiebert** [reply]
@scrozier Looks like there was a bug introduced, that is being fixed. I expect the next version will have it fixed.

https://github.com/remix-run/react-router/commit/174897bdd4d626066d67328773ef9b55610c04f9

`npm install react-router-dom-v5-compat@6.4.0` allowed me to install.
